api/index.js

```diff
@@ -87,9 +87,13 @@ var endpoint = config.api.protocol + '://' +
                config.api.host + ':' + config.api.port +
                '/api/v0/events/birth';
 
-var requestData = function requestData(url, callback) {
+var requestData = function requestData(url, user, callback) {
   return new Promise(function requestDataPromise(resolve, reject) {
-    return requestWithOAuth2.get(url, oauthUrl, clientId, clientSecret, oAuthUsername, oAuthPassword,
+    const headers = user ? {'X-Auth-Downstream-Username': user}: {};
+    return requestWithOAuth2.get({
+      'url': url,
+      'headers': headers
+    }, oauthUrl, clientId, clientSecret, oAuthUsername, oAuthPassword,
```

Comment:
I'm trying to follow the semantics of the request library, so requestWithOAuth2 can just pass that first argument straight through (after adding on the additional auth header anyway). The request library's get method either expects just a url string, or an object in the same format I've got above.
https://github.com/request/request